<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::TransferDialog</class>
 <widget class="QDialog" name="Otter::TransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opening unknown file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fromLabel">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="openWithLabel">
       <property name="text">
        <string>Open with:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="namelLayout">
       <item>
        <widget class="QLabel" name="iconLabel">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Otter::TextLabelWidget" name="nameTextLabelWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="Otter::TextLabelWidget" name="typeTextLabelWidget" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="Otter::TextLabelWidget" name="sizeTextLabelWidget" native="true"/>
     </item>
     <item row="3" column="1">
      <widget class="Otter::TextLabelWidget" name="fromTextLabelWidget" native="true"/>
     </item>
     <item row="4" column="1">
      <widget class="Otter::ApplicationComboBoxWidget" name="openWithComboBoxWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberChoiceCheckBox">
     <property name="text">
      <string>Remember choice for this file type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Otter::ApplicationComboBoxWidget</class>
   <extends>QComboBox</extends>
   <header>src/ui/ApplicationComboBoxWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
